runtime/lib/ttnn/operations/eltwise/unary/unary_composite.cpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+#include "unary_composite.h"
+#include "tt/runtime/detail/logger.h"
+#include "tt/runtime/detail/ttnn.h"
+#include "tt/runtime/ttnn/operations/eltwise/unary/utils.h"
+#include "tt/runtime/ttnn/operations/utils.h"
+#include "ttnn/operations/eltwise/unary/unary_composite.hpp"
+
+namespace tt::runtime::ttnn::operations::unary::composite {
+
+static void runEltwiseUnaryCompositeOP(
```

Comment:
What do we mean when we say "Composite" op in our runtime?

Reply:
It reflects the scheme of ttnn (in `src/tt-metal/ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp`). They have a concept of unaryCompositeOp that has a different execution function signature than ordinary unary ops.

Reply:
I see ok makes sense.